bin/plotting/pycbc_plot_singles_vs_params

```diff
@@ -109,7 +129,8 @@ y = y[mask]
 hexbin_style = {
     'gridsize': opts.grid_size,
     # hexbin shows bins with *less* than mincnt as blank
-    'mincnt': 0,
+    # (in earlier mpl versions this was less than *or equal to* mincnt)
+    'mincnt': 1 if matplotlib.__version__ >= '3.8.1' else 0,
     'linewidths': 0.03
 }
```

Comment:
Don't know why github won't let me write a suggestion here .. anyway, the suggestion is to remove the `mincnt` setting and comment line from this dict assignment and then write 
```
# In earlier versions mpl will try to take the max over bins with 0 triggers
# and fail, unless we tell it to leave these blank by setting mincnt
if matplotlib.__version__ >= '3.8.1':
    hexbin_style['mincnt'] = 0
```

Reply:
Updated as suggested (except with the `<=` sign the right way around)

Reply:
Github is obviously more awake than I am